tdrs-backend/wait_for_services.sh

```diff
@@ -1,7 +1,8 @@
 #!/usr/bin/env bash
 wait-for-it \
     --service http://clamav-rest:9000 \
-    --service http://localstack:4566 \
+    --service http://localstack:4566/health \
+    --timeout 60 \
```

Comment:
This increased timeout was necessary in CI contexts, where it takes a little longer to get everything started.